test/controllers/api/v2/locations_controller_test.rb

```diff
@@ -1,4 +1,5 @@
 require 'test_helper'
+require 'rfauxfactory'
```

Comment:
it would be better to add this to test_helper so every test can use this library, or just wait until https://github.com/theforeman/foreman/pull/5295 will add it and just remove this line after you rebase :-)